src/pages/Detailpage/Components/BadgesComponent.js

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+
+const BadgesComponent = (props) => {
+  const { detailLogo } = props;
```

Comment:
```suggestion
const BadgesComponent = ({ detailLogo }) => {
```
함수 파라미터 자리에서 직접 구조분해 할당 가능합니다

Reply:
와.. 이렇게 간단한 방법이... 바로 적용했습니다!